### PR TITLE
TGMC Christmas 2024

### PR DESCRIPTION
**Pull request by RipGrayson**
## About The Pull Request

Gemini_Generated_Image_qcr42lqcr42lqcr4

With Thanksgiving officially over, it's time to move onto the next holiday.
Introducing the TGMC Christmas PR, this PR

-Adds festive decorations to all ships and shuttles.
-Adds Christmas food and drink to all ships and shuttles.
-Places Christmas trees throughout the ships.
-Implements gifts with randomly generated presents inside, much like TG already does (for the sake of balance the items are much more curated compared to TG.)
-Allows players the magical chance to meet Santa and his elves face to face.
-Various other surprises and things I can't be bothered to note.

This PR isn't meant to be merged, only TMC'ed. At a later date I _might_ add some of the holiday stuff into the code itself for easier reuse in future Christmases.

## Why It's Good For The Game

We didn't have Christmas in 2021, I have taken it upon myself to fix this problem. If you don't like this PR you are officially a scrooge who needs to embark on a vaguely allegorical journey to learn the true meaning of Christmas.

## Changelog
:cl:
add: Added Christmas. If you're seeing this in game the PR was merged somehow, whoops.
/:cl:

### CODE REVIEW

**Comment by MLGTASTICa** [comment]
No mention of boiler lobbing gifts instead of globs. Christmas ruined.

File: code/game/objects/items/storage/pouch.dm [hunk 1]
```diff
@@ -553,7 +553,6 @@
 	. = ..()
 	storage_datum.sprite_slots = 1
 	storage_datum.max_w_class = WEIGHT_CLASS_BULKY
-	storage_datum.storage_slots = 7
```
Comment (ivanmixo) [inline code/game/objects/items/storage/pouch.dm:556]: ?

**Review by ivanmixo — CHANGES_REQUESTED** [review]

File: code/game/objects/items/storage/pouch.dm (hunk 1 above)
Comment (RipGrayson) [reply code/game/objects/items/storage/pouch.dm]: Accident, probably caused by copypasting code from old branch

**Review by TiviPlus — APPROVED** [review]

**Comment by RipGrayson** [comment]
It's not Christmas anymore

**Comment by RipGrayson** [comment]
Christmas is here again

**Pull request closed without merging**